src/components/calcite-stepper-item/calcite-stepper-item.tsx

```diff
@@ -0,0 +1,255 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop,
+  Watch
+} from "@stencil/core";
+import {
+  UP,
+  DOWN,
+  ENTER,
+  HOME,
+  END,
+  SPACE,
+  LEFT,
+  RIGHT
+} from "../../utils/keys";
+import { getElementDir, getElementProp } from "../../utils/dom";
+
+@Component({
+  tag: "calcite-stepper-item",
+  styleUrl: "calcite-stepper-item.scss",
+  shadow: true
+})
+export class CalciteStepperItem {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** is the step active */
+  @Prop({ reflect: true, mutable: true }) active: boolean = false;
+
+  /** has the step been completed */
+  @Prop({ reflect: true, mutable: true }) complete: boolean = false;
+
+  /** does the step contain an error that needs to be resolved by the user */
+  @Prop({ mutable: true }) error: boolean = false;
+
+  /** is the step disabled and not navigable to by a user */
+  @Prop({ mutable: true }) disabled: boolean = false;
+
+  /** pass a title for the stepper item */
+  @Prop() itemTitle?: string;
+
+  /** pass a title for the stepper item */
+  @Prop() itemSubtitle?: string;
+
+  // internal props inherited from wrapping calcite-stepper
+  /** pass a title for the stepper item */
+  /** @internal */
+
+  @Prop({ reflect: true, mutable: true }) layout?: string;
+
+  /** should the items display an icon based on status */
```

Comment:
you should be able to put `@internal` at the end in one line, or just use the multi-line jsdoc syntax

Reply:
Yeah these don't actually need to be docced, since they are just used internally...